client/lib/components/recent_numbers_graph.dart

```diff
@@ -40,6 +54,19 @@ class _RecentNumbersBarGraphState extends State<RecentNumbersBarGraph> {
           show: false,
         ),
         barTouchData: BarTouchData(
+          touchCallback: (touchResponse) {
+            setState(() {
+              swapDuration = Duration(microseconds: 1);
```

Comment:
Why 1000 Millisecs here and 750 above? Ideally the duration should be specified once only.

More importantly, why change the value at all? I’d expect there to be a single default and not to need changing it anywhere as you also do below.

https://en.m.wikipedia.org/wiki/Don%27t_repeat_yourself

Reply:
Sorry, misread that... it's 0.001 milliseconds for a microsecond. That's fine then.